Powershell-PoC-Environment-Setup.ps1

```diff
@@ -242,6 +432,8 @@ $rgStorage = New-AzureRmResourceGroup -Name "poc-storage-rg" -Location $eastLoca
 
 # Subnet Creation
 Write-Verbose "Creating Subnets..." -Verbose

```

Comment:
Does the Write-WithTime also outputs to console? If yes, please remove the duplicated Write-Verboses